### DISCUSSION TITLE
mlx model-zoo

### DISCUSSION THREAD

**Discussion by nsantavas** [ml-explore/mlx#242] [Ideas]
Hello everyone,

I was wondering whether you find it a good idea of having an mlx model-zoo similar to the torchvision models. I know that there is the [mlx-examples](https://github.com/ml-explore/mlx-examples), but I don't think that's the goal of this particular repo.

**Comment by awni** [comment]
Nice idea! Indeed that was not the original intention of the MLX examples though it is starting to head in that direction. It probably makes more sense to have a model zoo repo which ideally can download pretrained models (probably from Hugging Face) but also elsewhere.

**Reply by nsantavas** [reply]
I believe that if there is a chance to be planned in the near future, it will gain a lot of traction, and will make MLX broadly used. Moreover I believe that such a move can increase the number of people that are actively contributing in the MLX ecosystem. I also find it a good idea an interface that translates torchvision/timm/huggingface models to MLX. wdyt?
